### ISSUE TITLE
¿Alguien con alguna idea de Ruby on Rails?

### ISSUE THREAD

**Issue by JArandaIzquierdo** [iblancasa/miercolesgeek#5]

**Comment by demiurgosoft** [comment]
Odio ruby on rails pero si estamos hablando de nivel básico casi cualquiera que haya acabado PDOO y sepa manejarse con orientación a objetos podría preparar algo apañao

**Comment by angrykoala** [comment]
Nivel básico?

**Comment by JArandaIzquierdo** [comment]
Si, yo por lo menos no tengo ni una ligera idea @demiurgosoft

**Comment by angrykoala** [comment]
pues en realidad, se algo mas bien poco de ruby, pero nada de rails, de todas formas, si no aparece nadie en una semanita o asi, me empollo el manual en plan intensivo y a lo mejor puedo hacer algo (o otra persona que tenga ganas de añadir "Ruby on rails" a su curri-cúlo

**Comment by angrykoala** [comment]
@MarFerPra esta haciendo chorradicas en ruby on rails, así que le toca dar esta también

**Comment by JJ** [comment]
@MarFerPra ¿te animas entonces a dar la charla?

**Comment by marco-fp** [comment]
Si, por mi perfecto. ¿Alguna fecha en concreto?

**Comment by JJ** [comment]
Como es habitual, pon un Doodle.

**Comment by JJ** [comment]
O, @MarFerPra , elige directamente el primer miércoles que esté libre. Al final cuando a todo el mundo le viene bien es los miércoles, mañana o tarde.

**Comment by angrykoala** [comment]
@MarFerPra, 20 o 27 de abril estan libres (de momento)

**Comment by angrykoala** [comment]
nope, 27 esta ocupado http://www.meetup.com/Granada-Geek/events/227720189/
